pkg/reconciler/resolutionrequest/resolutionrequest.go

```diff
@@ -78,5 +80,5 @@ func requestDuration(rr *v1beta1.ResolutionRequest) time.Duration {
 }
 
 func timeoutMessage() string {
```

Comment:
I'd change this to take `timeout` as an argument, so we don't need the top-level `var timeout ...`.